docs/tasks/debug-application-cluster/audit.md

```diff
@@ -72,6 +72,24 @@ In both cases, audit events structure is defined by the API in the
 `audit.k8s.io` API group. The current version of the API is
 [`v1beta1`][auditing-api].
 
+**Note:** for example, in case of patches, request body is a JSON array with patch operations, not
```

Comment:
Nit, but do we need this first "for example"? (Starts the second sentence also)
if you omit, s/in/In

Reply:
Removed first `for example`. I think the second one is legitimate, since it indicates that an example follows